ablator/analysis/plot/cat_plot.py

```diff
@@ -16,18 +16,70 @@
 
 class Categorical(Plot):
     """
-    Class for categorical plots
+    This class is for preparing the results that are associated with each categorical attribute to be studied
+    (e.g., grouping metric results with each of the attributes). Its constructor takes in as input positional
+    arguments or keyword arguments from the base class `Plot`. Possible arguments are listed in the Parameters
+    section. The Attributes section lists its own attributes as well as those that are inherited.
+
+    Parameters
+    ----------
+    *args : ty.Any
```

Comment:
Is this standard way of documenting it? I believe it is better to simply `pretend` as if the arguments are those of `Categorical` and document them as such and we can add a flake8 `noqa` to ignore the error.

Reply:
Originally I was simply pretend as if the arguments are those of Categorical and document them as such, but that created flake8 errors which says `Categorical` parameters does not match docstring parameters. That's why I changed it to the current version

Reply:
Let's remove *args and **kwargs and document each parameter as if it belongs to `Categorical`.